docs/REVIEWING.md

```diff
@@ -70,8 +78,11 @@ documentation, nor should they be merged with "TODOs" opened for documentation.
 Vector currently offers 2 methods for performance testing:
 
 1. Internal benchmarks located in the [`/benches` folder](benches).
-2. A full end-to-end [test harness](https://github.com/vectordotdev/vector-test-harness)
-   for complex integration and performance testing.
+2. A full end-to-end [soak test
+   suite](https://github.com/vectordotdev/vector/tree/master/soaks) for complex
+   integration and performance testing.
+
+For new integrations, consider whether the a new soak test should be added.
```

Comment:
```suggestion
For new integrations, consider whether a new soak test should be added.
```

I'm not sure if this advice is particularly useful. The soak tests are of primary importance for those paths that we want to ensure stay performant, and new integrations would not likely be one of those.